intelij-plugin/src/main/resources/META-INF/plugin.xml

```diff
@@ -15,19 +15,39 @@
     <version>1.0</version>
```

Comment:
Is this the version of plugin? Are we jumping from 1.0 to 1.2?

Reply:
this whole property is driven by gradle version of project (it is injected into plugin.xml during build) so this property should be removed from this file, as it is just confusing and not needed